html/videoeditor.html

```diff
@@ -58,7 +58,7 @@ <h2>Add the finishing touches to your video.</h2>
             <div id="reset">Reset changes (revert to initial recording)</div>
             <label id="filename-select-label" for="filename-select">File name</label>
             <select id="filename-select">
-                <option value="random">Random filename (default)</option>
```

Comment:
I think that this "filename format" choose in unecessary.  Could always be in ISO format as you implemented.

But, if the choose remains, I recommend label internationalization (using i18n) for this new labels.